modules/bibcirculation/lib/bibcirculation_dblayer.py

```diff
@@ -591,7 +591,46 @@ def get_pdf_request_data(status):
 ###
 
 
+def calculate(s, e, n):
```

Comment:
I think you shouldn't use one character variables, characters are for free :stuck_out_tongue_winking_eye: